### ISSUE TITLE
Pagination page-size select menu is too narrow on IE & Edge

### ISSUE THREAD

**Issue by wmitchell-cambria** [ca-cwds/design-system#357]
<img width="559" alt="pagination in edge" src="https://user-images.githubusercontent.com/36680514/50921908-f9327d00-13fd-11e9-907d-63e36299ba2d.png">

I'll try to get a PR together, but let me know if you have any immediate ideas, @eckdanny.

**Comment by wmitchell-cambria** [comment]
Do you think it would be feasible to style these without flex box? It seems like they are just a set of centered inline-block elements. Flexbox may be overkill, and it seems to be the source of the extra padding and/or tiny select menu.

**Comment by eckdanny-osi** [comment]
@wmitchell-cambria related to https://github.com/ca-cwds/design-system/issues/333

Flexbox is used in the default `Pagination` subcomponent of `react-table`. Agree its overkill for our needs. This is up for grabs.

**Comment by wmitchell-cambria** [comment]
I was bothered by the fact that it was working OK in Edge on Netlify and not in CANS. Turns out we were setting width in a couple places, and we need to also set min-width for IE and Edge to recognize it. Turns out there are a handful of similar [known issues](https://caniuse.com/#feat=flexbox) on Can I Use.

Since the widths we were setting were due to React Wood Duck rem issues, for now I just modified those overrides. So...no PR incoming at the moment. I think let's close this, because I agree that the real solution is #333.
